scripts/zones/Horlais_Peak/mobs/Aries.lua

```diff
@@ -0,0 +1,216 @@
+-----------------------------------
+-- Area: Horlais Peak
+--  Mob: Aries
+-- BCNM: Today's Horoscope
+-----------------------------------
+local ID = zones[xi.zone.HORLAIS_PEAK]
+-----------------------------------
+---@type TMobEntity
+local entity = {}
+
+-- TODO: Ninjutsu resists
+-- TODO: Song resists
+
+local regenPower     = 100
+local skillWeights   =
+{
+    { id = xi.mobSkill.PETRIBREATH, chance = 40 },
+    { id = xi.mobSkill.GREAT_BLEAT, chance = 35 },
+    { id = xi.mobSkill.RUMBLE,      chance = 10 },
+    { id = xi.mobSkill.RAGE_2,      chance = 10 },
+    { id = xi.mobSkill.RAM_CHARGE,  chance = 5  },
+}
+
+-----------------------------------
+-- Checks if the mob is asleep.
+-----------------------------------
+local isMobAsleep = function(mob)
+    return
+        mob:getStatusEffect(xi.effect.SLEEP_I) or
+        mob:getStatusEffect(xi.effect.SLEEP_II) or
+        mob:getStatusEffect(xi.effect.LULLABY)
+end
+
+-----------------------------------
+-- Used to block the self-sleep action while taking other actions.
+-----------------------------------
+local abilityBlocked = function(mob)
+    local action = mob:getCurrentAction()
+
+    return
+        action == xi.act.MOBABILITY_START or
+        action == xi.act.MOBABILITY_USING or
+        action == xi.act.MOBABILITY_FINISH or
+        action == xi.act.MAGIC_START or
+        action == xi.act.MAGIC_CASTING or
+        action == xi.act.MAGIC_FINISH
+end
```

Comment:
This function idk if we have already. But if we dont, I think its a prime candidate for an util global function